examples/fluids/vortexshedding.yaml

```diff
@@ -0,0 +1,55 @@
+problem: 'vortexshedding'
+
+# Time Stepping Settings
+implicit: true
+stab: supg
+
+output_freq: 10
+
+ts:
+  adapt_type: 'none'
+  type: alpha
+  dt: .05
+  max_steps: 2000
+  alpha_radius: 0.5
+
+freestream:
+  velocity: 1,0,0
+  temperature: 24.92
+  pressure: 7143
+g: 0,-.01,0
+
+# viscosity
+mu: 0.01
+
+# Cylinder diameter
+D:  1.41421356
+
+## DM Settings:
+degree: 2
+dm_plex_dim: 3
+dm_plex_box_lower: 0,0,0
+dm_plex_box_upper: 1,1,0.1
+dm_plex_box_faces: 50,20,1
+dm_plex_filename: examples/fluids/cylinder.msh
+
+# Boundary Settings
+bc_slip_z: 6
+bc_wall: 5
+bc_freestream: 1,2
+bc_slip_y: 3,4
+wall_comps: 1,2,3
+
+# TODO: fix code to convert to the state variables being used
```

Comment:
Reminder :point_up:

Reply:
So this is the initial condition I have which should go zero at the cylinder walls
```
CeedPragmaSIMD for (CeedInt i = 0; i < Q; i++) {
    CeedScalar       U[5]  = {0.};
    CeedScalar       qi[5] = {0.};
    const CeedScalar x[3]  = {X[0][i], X[1][i], X[2][i]};

    U[0] = S_infty.U.density;
    U[1] = U[0] * tanh(Min(0, fabs(x[0] - center) - radius) / D);
    U[2] = U[0] * tanh(Min(0, fabs(x[1] - center) - radius) / D);
    U[3] = U[0] * tanh(Min(0, fabs(x[2] - center) - radius) / D);
    U[4] = S_infty.Y.pressure / (gamma - 1) + e_kinetic;

    State InitCond = StateFromU(newtonian_ctx, U, x);
    StateToQi(newtonian_ctx, InitCond, qi);

    for (CeedInt j = 0; j < 5; j++) q0[j][i] = qi[j];

  }
```
This initial condition should be converted to conservative variables?